src/core/include/ngraph/opsets/opset.hpp

```diff
@@ -47,18 +47,19 @@ class NGRAPH_API OpSet : public ov::OpSet {
         ov::OpSet::insert<OP_TYPE>(OP_TYPE::get_type_info_static().name);
     }
 
-    ngraph::FactoryRegistry<ngraph::Node>& get_factory_registry() {
-        return m_factory_registry;
-    }
+    ngraph::FactoryRegistry<ngraph::Node> get_factory_registry();
```

Comment:
`const ngraph::FactoryRegistry<ngraph::Node>& get_factory_registry() const;`

Reply:
Thanks, will be applied, but last `const` isn't applicable because I fill `m_factory_registry` with `m_factory` values